schema/data/anchored/v0/classes/BT01.jsonld

```diff
@@ -1,96 +1,52 @@
-        }
+      "rfs:impactType": "IMPROVED_BIODIVERSITY",
+      "rfs:supportsSDG": [
+        "SDG_13"
       ],
       "schema:name": "Climate"
     },
     {
-      "@id": "regen:WildlifeHabitat-Terrasos",
-      "@type": "regen:ProjectBenefit",
-      "schema:name": "Wildlife Habitat"
+      "rfs:impactType": "IMPROVED_WILDLIFE_HABITAT",
+      "schema:name": "Improved Wildlife Habitat"
     }
   ],
   "schema:description": "The Biodioversity Credits protocol outlines a strategic framework for the development of projects that guarantee quantifiable gains in biodiversity, all while providing robust financial and legal assurances to secure their sustainability and enduring impact. This pioneering approach to biodiversity conservation and resource management emphasizes the selection of projects distinguished by their significant ecological value within a performance-based model. \n This protocol streamlines the process of registration, quantification, and issuance of Biodiversity Credits (VBC) for conservation projects that meet the eligibility criteria. It is designed to resonate with both individuals and organizations committed to contributing to the preservation of threatened ecosystems and biodiversity, ultimately delivering long-term sustainability and socio-economic benefits.",
-  "regen:creditingTerm": "P30Y",
-  "regen:ecosystemType": [
-    "Terrestrial"
+  "rfs:hasCreditingPeriod": "P30Y",
+  "rfs:eligibleEnvironmentTypes": [
+    "rft:Terrestrial"
   ],
-  "regen:primaryImpact": {
-    "@id": "regen:Biodiversity-Terrasos",
-    "regen:SDGs": [
-      {
-        "@id": "regen:15LifeOnLand"
-      }
+  "rfs:hasPrimaryImpact": {
+    "rfs:impactType": "IMPROVED_BIODIVERSITY",
+    "rfs:supportsSDG": [
+      "SDG_15"
     ],
-    "schema:name": "Biodiversity"
+    "schema:name": "Improved Biodiversity"
   },
-  "regen:sourceRegistry": {
-    "schema:url": "https://en.terrasos.co/",
-    "schema:name": "Terrasos"
-  },
-  "regen:permanencePeriod": "P20Y",
-  "regen:eligibleActivities": [
-    "Ecosystem Preservation",
-    "Forest Enhancement",
-    "Ecological Restoration"
-  ],
-  "regen:approvedMethodologies": {
-    "@type": "schema:ItemList",
-    "schema:itemListElement": [
-      {
-        "schema:url": "https://www.registry.regen.network/crediting-protocols/terrasos-biodiversity-unit",
-        "schema:name": "Terrasos Protocol for Issuing Voluntary Biodiversity Credits",
-        "schema:version": "V3.0",
-        "schema:identifier": ""
-      },
-      {
-        "schema:url": "https://www.registry.regen.network/crediting-protocols/terrasos-biodiversity-unit",
-        "schema:name": "Terrasos Protocol for Issuing Voluntary Biodiversity Credits",
-        "schema:version": "V4.0",
-        "schema:identifier": ""
-      }
-    ]
-  },
-  "regen:creditGenerationMethod": [
-    "Voluntary Biodiversity Credits"
+  "rfs:eligibleActivities": [
+    "CONSERVATION",
+    "AFFORESTATION",
+    "ECOSYSTEM_RESTORATION"
```

Comment:
we can change this to ENVIRONMENT_RESTORATION

Reply:
What about "ECOLOGICAL_RESTORATION" ?  "Environment restoration" is not a term I can find as many references for externally, so think this might be better?

https://en.wikipedia.org/wiki/Ecological_restoration

Also- can this be done in a follow-up PR?